packages/remote-config/ios/RNFBConfig/RNFBConfigModule.m

```diff
@@ -223,6 +247,41 @@ + (BOOL)requiresMainQueueSetup {
   }
 }
 
+RCT_EXPORT_METHOD(onConfigUpdated
+                  : (FIRApp *)firebaseApp
+                  : (RCTResponseSenderBlock)callback) {
+    FIRConfigUpdateListenerRegistration *newRegistration = [[FIRRemoteConfig remoteConfigWithApp:firebaseApp] addOnConfigUpdateListener:^( FIRRemoteConfigUpdate * _Nonnull configUpdate, NSError * _Nullable error) {
+        NSMutableDictionary *userInfo;
+
+        if (error != nil) {
+            userInfo = [NSMutableDictionary dictionary];
+
+            [userInfo setValue:@(NO) forKey:@"fatal"];
+            [userInfo setValue:@"unknown" forKey:@"code"];
```

Comment:
We should probably try to propagate the original error code `error.code` before making it defacto `unknown`.